tests/test_utils.py

```diff
@@ -176,12 +176,12 @@ def test_get_pr_text(self):
         """Test get PR text"""
         tmp = sys.argv
         sys.argv = ['a', 'b', 'c', 'd']
-        expected = 'To reproduce this PR, run the following command.\n\n'\
+        expected = 'This pull request was generated by issuing:\n\n'\
                    '```\na b c d\n```\n'
         self.assertEqual(expected, get_pr_text())
         # test with an argument
         expected = 'sample\n'\
-                   'To reproduce this PR, run the following command.\n\n'\
+                   'This pull request was generated by issuing:\n\n'\
```

Comment:
@andre-rosa could you change this to "This pull request was generated by running the following command."?

Reply:
Yeah issuing is not a clear verb here.

```suggestion
                   'This pull request was generated by running the following command:\n\n'\
```

Reply:
Thank you both! I had also to change the test case, done in latest patchset.